source/CIGameScene.h

```diff
@@ -81,7 +87,7 @@ class GameScene : public cugl::Scene2 {
      * This constructor does not allocate any objects or start the game.
      * This allows us to use the object without a heap pointer.
      */
-    GameScene() : cugl::Scene2() {
+    GameScene() : cugl::Scene2(), _spawnRate(40.0f), _colorCount(6), _countdown(-10.0f) {
```

Comment:
You can put these constants in the header file and they'll also be included in the cpp file. i.e. BASE_SPAWN_RATE and INACTIVE_COUNTDOWN should be in this file

Reply:
Should I create a header file for all the default/constant values in menu scenes? It would essentially be a namespace with all the menu constants defined.

Reply:
I don't think that's necessary b/c there aren't too many constants. We may want a global constants file at some point but that's definitely not needed for this pr